xml/System.Security.Cryptography/HMAC.xml

```diff
@@ -346,9 +346,20 @@
         <Parameter Name="source" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="source">The input to compute the HMAC code for.</param>
+        <summary>Routes data written to the object into the HMAC algorithm for computing the HMAC.</summary>
```

Comment:
Style guide rule for always throwing exception:
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Method

```suggestion
        <summary>Throws a <see cref="T:System.PlatformNotSupportedException" /> in all cases.</summary>
```

Reply:
Yeah, I thought about this, but since it's virtual and the guidance is "override it so it doesn't throw", it felt misleading to me to say "in all cases".  But I'm easily swayed here.

Reply:
yes I am too and was a bit on the fence on these. You can leave as is 😄